index.js

```diff
@@ -11,16 +11,19 @@ const defaults = {
 exports.register = (server, options, next) => {
   options = Object.assign({}, defaults, options);
 
-  server.ext('onPreHandler', (request, reply) => {
-    if (request.method !== 'get') {
+  server.ext('onPostHandler', (request, reply) => {
```

Comment:
problem with doing onPostHandler, if we call the method on their first hit, where they have a referrer, but no cookie, it won't have anything.  If it's preHandler, it should, right?

Reply:
If its preHandler we wont get if its a view or not.

Reply:
Perhaps a better solution would be ignore the source and just block off paths

Reply:
yeah, lets blacklist routes